scrapers/watchingmydaughtergoblack.yml

```diff
@@ -0,0 +1,40 @@
+name: "watchingmydaughtergoblack"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - watchingmydaughtergoblack.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h3[@class="scene-title col-xs-12"]/text()
+      Date:
+        selector: //meta[@itemprop="uploadDate"]/@content
+        postProcess:
+          - replace:
+              - regex: ([\d-]*).+
+                with: $1
+          - parseDate: 2006-01-02
+      Details:
+        selector: //p[@itemprop="description"]/text()
+      Tags:
+        Name: 
+          selector: //p[@class="categories"]/text()
+          postProcess:
+            - replace:
+              - regex: ',(\x{00A0}|\s)+'
+                with: ","
+          split: ","
+      Performers:
+        Name: //span[@class="starring-list"]/a/text()
+      Image:
+        selector: //meta[@itemprop="thumbnailUrl"]/@content
+        postProcess:
+          - replace:
+              - regex: ^
+                with: "https:"
+      Studio:
+        Name:
+          fixed: "WatchingMyDaughterGoBlack"
```

Comment:
I used the url of the thumb to get the actual studios
```yaml
      Studio:
        Name:
          selector: //meta[@itemprop="thumbnailUrl"]/@content
          postProcess:
            - replace:
                - regex: '^https://(?:[^\.]+\.)?([^\./]+).+'
                  with: $1
            - map:
                blackmeatwhitefeet: Black Meat White Feet
                blacksonblondes: Blacks On Blondes
                cuckoldsessions: Cuckold Sessions
                gloryhole-initiations: Gloryhole-Initiations
                gloryhole: Glory Hole
                interracialblowbang: Interracial Blow Bang
                katiethomas: Katie Thomas
                ruthblackwell: Ruth Blackwell
                springthomas: Spring Thomas
                watchingmydaughtergoblack: Watching My Daughter Go Black
                watchingmymomgoblack: Watching My Mom Go Black
                wefuckblackgirls: We Fuck Black Girls
                wifewriting: Wife Writing
                zebragirls: Zebra Girls
                #below not tested
                barbcummings: Barb Cummings
                blacksoncougars: Blacks On Cougars
                candymonroe: Candy Monroe
                dogfartbehindthescenes: Dogfart Behind The Scenes
                interracialpickups: Interracial Pickups
                theminion: The Minion
```
the last 6 i couldnt find a scene easily in the site to double check